MVP/MVP.csproj

```diff
@@ -47,7 +47,6 @@
         <Folder Include="Models\Enums\" />
         <Folder Include="Services\Demo\" />
         <Folder Include="Services\Demo\Files\" />
-        <Folder Include="Resources\" />
```

Comment:
Why are you removing my Resources folder :(

Reply:
🤷‍♂️ I will ask the IDE responsible for that...